lifecycle_test_library/scripts/lifecycle_node_test.py

```diff
@@ -53,6 +53,7 @@
     #Start the test
     test_obj = LifecycleTestLibrary(topic, publications, publishing_hz, subscriptions, services)
     
+    # TODO(lucasw) neither of these work
```

Comment:
Is this still the case? Running `roslaunch lifecycle_test_library lifecycle_test.launch` in this branch prints out

```
...
ROS_MASTER_URI=http://3_xterm:11311                                                                                           [17/55]setting /run_id to 835f9f86-b08b-11eb-833e-0242c0a8f007process[rosout-1]: started with pid [4915]started core service [/rosout]
process[Test_Node-2]: started with pid [4924]
process[Example_Node-3]: started with pid [4933]
Hi Welcome to the lifecycle node testing tool!
Start Tests
Welcome to the example node
_on_configure
_on_cleanup
_on_configure
_on_activate
_on_deactivate
_on_cleanup
_on_configure
_on_activate
_on_deactivate
_on_cleanup
_on_cleanup
Test UNCONFIGURED
Test INACTIVE
_on_configure
Test ACTIVE
_on_activate
Test FINALIZED
_on_shutdown
[WARN] [1620539899.143230]: Could not process inbound connection: [/Example_Node] is not a publisher of [/Example_Node/chatter]. Topics are [['/rosout', 'rosgraph_msgs/Log'], ['/Example_Node/lifecycle_state', 'lifecycle_msgs/Lifecycle'], ['/Example_Node/lifecycle/result', 'lifecycle_msgs/LifecycleActionResult'], ['/lm_events', 'lifecycle_msgs/lm_events'], ['/Example_Node/lifecycle/feedback', 'lifecycle_msgs/LifecycleActionFeedback'], ['/Example_Node/lifecycle/status', 'actionlib_msgs/GoalStatusArray']]{'message_definition': '', 'callerid': '/Test_Node', 'tcp_nodelay': '0', 'md5sum': '*', 'topic': '/Example_Node/chatter', 'type': '*'}
Node tested successfully
================================================================================REQUIRED process [Test_Node-2] has died!
process has finished cleanly
log file: /home/user/.ros/log/835f9f86-b08b-11eb-833e-0242c0a8f007/Test_Node-2*.log
Initiating shutdown!
================================================================================
[Example_Node-3] killing on exit
[Test_Node-2] killing on exit
[rosout-1] killing on exit
[master] killing on exit
shutting down processing monitor...
... shutting down processing monitor complete
done
```

Seeing `Node tested successfully` makes me believe that asserts were successfully ran?

Reply:
I didn't realize that `managed_node_example.py` had to be running along with the launch file, but later added it to the lifecycle_test.launch and that caused the test to work, removed out-dated comment now.